model/database/common.yaml

```diff
@@ -60,6 +64,35 @@ groups:
       - ref: db.cosmosdb.consistency_level
         requirement_level:
           conditionally_required: If available.
+      - ref: error.type
+        examples: ["CosmosException", "CosmosOperationCanceledException", "CosmosObjectDisposedException", "CosmosNullReferenceException"]
```

Comment:
it should be fully qualified exception name `Microsoft.Azure.Cosmos.CosmosOperationCanceledException`